### PR TITLE
Adds zi'pluax mantle, slugger wetsuit to Skrell loadout + minor bugfix

### PR DESCRIPTION
**Pull request by Llywelwyn**
rscadd: "Added zi'pluax mantle and prescient republic of the qar wetsuit to Skrell loadout."
bugfix: "Adds missing sprites when some Skrell items were equipped on ears."

image

### CODE REVIEW

**Comment by Llywelwyn** [comment]
!review

File: code/modules/clothing/head/xenos/skrell.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/obj/item/clothing/head/skrell
```
Comment (SleepyGemmy) [inline code/modules/clothing/head/xenos/skrell.dm:1]: you need to check this new file in the .dme.

**Review by Alberyk — APPROVED** [review]

**Comment by Llywelwyn** [comment]
ill fix this shit once #13717 gets merged i figured out how

i hate sprites and dmis im never doing work for ryver again

**Comment by Llywelwyn** [comment]
!wip

**Comment by Llywelwyn** [comment]
!review

**Comment by Llywelwyn** [comment]
!bugfix

**Pull request merged**